charts/camunda-platform/charts/operate/templates/deployment.yaml

```diff
@@ -44,6 +44,8 @@ spec:
             value: {{ include "camundaPlatform.issuerBackendUrl" . | quote }}
           - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI
             value: {{ printf "%s%s" (include "camundaPlatform.issuerBackendUrl" .) "/protocol/openid-connect/certs" | quote }}
+          - name: CAMUNDA_OPERATE_IDENTITY_BASEURL
+            value: "http://{{ .Release.Name }}-identity:80"
```

Comment:
I believe that the [context path](https://github.com/camunda/camunda-platform-helm/blob/fcf6756d9f8ed6096881a376302d4e39c3764b74/charts/camunda-platform/values.yaml#L1379) var should be added.

Also since the same var used in different places, it's better to make it a template in [_helpers.tpl](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/templates/_helpers.tpl)